Traceback (most recent call last):
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/asyncio/base_events.py", line 650, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/contextlib.py", line 222, in __aexit__
    await self.gen.athrow(typ, value, traceback)
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/nbclient/client.py", line 608, in async_setup_kernel
    yield
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/python-lll/miniconda3/envs/bb1000/lib/python3.11/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
%%bash
mkdir MainFolder
cd MainFolder
mkdir NewFolder
cd NewFolder
pwd > introduction.txt
echo introduction.txt >> introduction.txt
mkdir LastFolder
cd LastFolder
pwd > data.txt
echo data.txt >> data.txt
cd ../..
pwd > Readme.txt
echo Readme.txt >> Readme.txt
mkdir SecondFolder
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mCalledProcessError[0m                        Traceback (most recent call last)
Cell [0;32mIn[1], line 1[0m
[0;32m----> 1[0m get_ipython()[38;5;241m.[39mrun_cell_magic([38;5;124m'[39m[38;5;124mbash[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124mmkdir MainFolder[39m[38;5;130;01m\n[39;00m[38;5;124mcd MainFolder[39m[38;5;130;01m\n[39;00m[38;5;124mmkdir NewFolder[39m[38;5;130;01m\n[39;00m[38;5;124mcd NewFolder[39m[38;5;130;01m\n[39;00m[38;5;124mpwd > introduction.txt[39m[38;5;130;01m\n[39;00m[38;5;124mecho introduction.txt >> introduction.txt[39m[38;5;130;01m\n[39;00m[38;5;124mmkdir LastFolder[39m[38;5;130;01m\n[39;00m[38;5;124mcd LastFolder[39m[38;5;130;01m\n[39;00m[38;5;124mpwd > data.txt[39m[38;5;130;01m\n[39;00m[38;5;124mecho data.txt >> data.txt[39m[38;5;130;01m\n[39;00m[38;5;124mcd ../..[39m[38;5;130;01m\n[39;00m[38;5;124mpwd > Readme.txt[39m[38;5;130;01m\n[39;00m[38;5;124mecho Readme.txt >> Readme.txt[39m[38;5;130;01m\n[39;00m[38;5;124mmkdir SecondFolder[39m[38;5;130;01m\n[39;00m[38;5;124m'[39m)

File [0;32m~/miniconda3/envs/bb1000/lib/python3.11/site-packages/IPython/core/interactiveshell.py:2430[0m, in [0;36mInteractiveShell.run_cell_magic[0;34m(self, magic_name, line, cell)[0m
[1;32m   2428[0m [38;5;28;01mwith[39;00m [38;5;28mself[39m[38;5;241m.[39mbuiltin_trap:
[1;32m   2429[0m     args [38;5;241m=[39m (magic_arg_s, cell)
[0;32m-> 2430[0m     result [38;5;241m=[39m fn([38;5;241m*[39margs, [38;5;241m*[39m[38;5;241m*[39mkwargs)
[1;32m   2432[0m [38;5;66;03m# The code below prevents the output from being displayed[39;00m
[1;32m   2433[0m [38;5;66;03m# when using magics with decodator @output_can_be_silenced[39;00m
[1;32m   2434[0m [38;5;66;03m# when the last Python token in the expression is a ';'.[39;00m
[1;32m   2435[0m [38;5;28;01mif[39;00m [38;5;28mgetattr[39m(fn, magic[38;5;241m.[39mMAGIC_OUTPUT_CAN_BE_SILENCED, [38;5;28;01mFalse[39;00m):

File [0;32m~/miniconda3/envs/bb1000/lib/python3.11/site-packages/IPython/core/magics/script.py:153[0m, in [0;36mScriptMagics._make_script_magic.<locals>.named_script_magic[0;34m(line, cell)[0m
[1;32m    151[0m [38;5;28;01melse[39;00m:
[1;32m    152[0m     line [38;5;241m=[39m script
[0;32m--> 153[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39mshebang(line, cell)

File [0;32m~/miniconda3/envs/bb1000/lib/python3.11/site-packages/IPython/core/magics/script.py:305[0m, in [0;36mScriptMagics.shebang[0;34m(self, line, cell)[0m
[1;32m    300[0m [38;5;28;01mif[39;00m args[38;5;241m.[39mraise_error [38;5;129;01mand[39;00m p[38;5;241m.[39mreturncode [38;5;241m!=[39m [38;5;241m0[39m:
[1;32m    301[0m     [38;5;66;03m# If we get here and p.returncode is still None, we must have[39;00m
[1;32m    302[0m     [38;5;66;03m# killed it but not yet seen its return code. We don't wait for it,[39;00m
[1;32m    303[0m     [38;5;66;03m# in case it's stuck in uninterruptible sleep. -9 = SIGKILL[39;00m
[1;32m    304[0m     rc [38;5;241m=[39m p[38;5;241m.[39mreturncode [38;5;129;01mor[39;00m [38;5;241m-[39m[38;5;241m9[39m
[0;32m--> 305[0m     [38;5;28;01mraise[39;00m CalledProcessError(rc, cell)

[0;31mCalledProcessError[0m: Command 'b'mkdir MainFolder\ncd MainFolder\nmkdir NewFolder\ncd NewFolder\npwd > introduction.txt\necho introduction.txt >> introduction.txt\nmkdir LastFolder\ncd LastFolder\npwd > data.txt\necho data.txt >> data.txt\ncd ../..\npwd > Readme.txt\necho Readme.txt >> Readme.txt\nmkdir SecondFolder\n'' returned non-zero exit status 1.
CalledProcessError: Command 'b'mkdir MainFolder\ncd MainFolder\nmkdir NewFolder\ncd NewFolder\npwd > introduction.txt\necho introduction.txt >> introduction.txt\nmkdir LastFolder\ncd LastFolder\npwd > data.txt\necho data.txt >> data.txt\ncd ../..\npwd > Readme.txt\necho Readme.txt >> Readme.txt\nmkdir SecondFolder\n'' returned non-zero exit status 1.

